sphinx/source/docs/user_guide/compat.rst

```diff
@@ -75,15 +60,25 @@ requires only one line of code in HoloViews:
  :align: center
 
 Adding overlaid plots, slider widgets, selector widgets, selection
-tools, and tabs is similarly straightforward. HoloViews objects can
-also be rendered using a Matplotlib-based backend, which allows SVG or
-PDF output not currently available for native Bokeh plots. See the
-Holoviews `Bokeh_Backend`_ tutorial for more details.
+tools, and tabs is similarly straightforward.
+
+Further synergy
+---------------
+
+You can render HoloViews objects with the `Matplotlib`_ library. This
+enables SVG and PDF output not currently available for native Bokeh
```

Comment:
@bryevdv Is this still correct, given the improved SVG export in Bokeh itself?

Reply:
Also, in case this sentence remains in here: The Google style guide recommends avoiding potentially socially-charged terms for technical concepts where possible (https://developers.google.com/style/inclusive-documentation#features-and-users). So, instead of "native", I'd recommend saying something like "not available in Bokeh itself."

Reply:
I think this comment is just out of date and can be removed